ts/store/reducers/backendStatus.ts

```diff
@@ -552,3 +553,27 @@ const sectionStatusUncachedSelector = (
     state.backendStatus.status,
     O.chainNullableK(status => status.sections[sectionStatusKey])
   );
+
+const statusMessagesSelector = createSelector(
```

Comment:
I do not think that this `createSelector` is needed.

`backendStatusSelector` returns a reference to an `Option`. Such reference is stored into redux state and is changed every time that the `backedStatus` property is updated (from the reducer). From such instance, the internal `statusMessages` instance is returned (or better said the reference to it).

So to summarise:
- if `backendStatus` is not updated, the same (pointer to the) instance is returned every time the selector is called.
- if `backendStatus` is updated, a new instance is returned (and after that, the condition above applies)

This behaviour is totally unaffected from the usage of `createSelector`.